core/src/main/resources/defaults/blueprints/cdp-data-engineering-702.bp

```diff
@@ -251,6 +245,32 @@
           }
         ]
       },
+      {
+        "refName": "das",
+        "serviceType": "DAS",
+        "roleConfigGroups": [
+          {
+            "refName": "das-DAS_WEBAPP",
+            "roleType": "DAS_WEBAPP",
+            "base": true,
+            "configs": [
+              {
+                "name": "data_analytics_studio_user_authentication",
```

Comment:
I think we should add this to the configprovider as well because probably every cluster which using DAS should include this property. cc @horadla23

Reply:
Moved.